packages/hydrooj/src/upgrade.ts

```
@@ -617,4 +617,27 @@ export const coreScripts: MigrationScript[] = [
     },
+    async function _95_96() {
+        await iterateAllDomain(async ({ _id }) => {
+            logger.info('Processing domain %s', _id);
+            const tdocs = await contest.getMulti(_id, {}).toArray();
+            for (const tdoc of tdocs) {
+                await contest.edit(_id, tdoc._id, {
+                    problems: tdoc.pids.map((pid, idx) => ({
+                        pid,
+                        label: getAlphabeticId(idx),
+                        ...(tdoc?.score && tdoc.score[pid] ? { score: tdoc.score[pid] } : {}),
+                        ...(tdoc?.balloon && tdoc.balloon[pid] ? {
```

Biome (1.9.4)

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)